concepticondata/concepticon.tsv

```diff
@@ -2994,7 +2994,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2998	DAMP	Sense perception	Refers to the presence of a liquid, especially water, often in trace amounts.	Property	
 2999	EVERYWHERE	Spatial relations	In all places.	Other	
 3000	LIVING	The body	Being alive.	Property	
-3001	POWERFUL	Emotions and values	Having great power.	Property	
+3001	POWERFUL	Emotions and values	Being done using a huge amount of energy.	Property	
```

Comment:
Thinking from elicitations and other concept list, the original seems closer to most intentions. This usually refers to a human (leader, guide) or to a supernatural being which has a lot of power, correct?

Reply:
Only a single list is linked to powerful: https://concepticon.clld.org/values/Dellert-2017-1016-537